app/Http/Controllers/UserListsController.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\UserList; // Ensure the correct model is imported
+use Illuminate\Http\Request;
+
+class UserListsController extends Controller
+{
+    public function allUsers()
+    {
+        $myUsers = UserList::all();
+
+        if ($myUsers->isEmpty()) {
+            return response()->json([
+                'success' => false,
+                'message' => 'No users found'
+            ]);
+        }
+
+        return response()->json([
+            'success' => true,
+            'data' => $myUsers
+        ]);
+    }
+
+    public function login(Request $request)
+    {
+        $request->validate([
+            'username' => 'required|string|max:255',
+            'password' => 'required|string|max:225',
+        ]);
+    
+        $username = $request->input('username');
+        $password = $request->input('password');
+    
+        $user = UserList::where('username', $username)->first();
+    
+        if (!$user) {
+            return response()->json([
+                'success' => false,
+                'message' => 'No items found for this user'
+            ], 404);
+        }
+    
+        if ($user->password !== $password) {
```

Comment:
password not encrypted?

Reply:
> password not encrypted?
here is the updated code for the login
if (!$user || !Hash::check($request->password, $user->password)) {
            return response()->json([
                'message' => 'Invalid login details'
            ], 401);
        }

Reply:
Please update PR.